linting/eslint.config.js

```diff
@@ -23,7 +13,7 @@ module.exports = {
     browser: true,
     node: true,
     es6: true,
-    'jest/globals': true,
+    jest: true,
```

Comment:
I'm guessing the `/globals` part isn't needed for eslint to be happy with jest code?